var/spack/repos/builtin/packages/amrex/package.py

```diff
@@ -136,9 +149,20 @@ class Amrex(CMakePackage, CudaPackage, ROCmPackage):
 
     # Build dependencies
     depends_on("mpi", when="+mpi")
-    depends_on("sundials@4.0.0:4.1.0 +ARKODE +CVODE", when="@19.08:20.11 +sundials")
-    depends_on("sundials@5.7.0: +ARKODE +CVODE", when="@21.07:22.04 +sundials")
-    depends_on("sundials@6.0.0: +ARKODE +CVODE", when="@22.05: +sundials")
+    with when("+ascent"):
+        depends_on("ascent")
+        depends_on("ascent +cuda", when="+cuda")
+        depends_on("ascent +mpi", when="+mpi")
+    with when("+catalyst"):
```

Comment:
Is this supposed to be:

```suggestion
    with when("+conduit"):
```